src/main/java/kitchenpos/order/domain/Order.java

```diff
@@ -0,0 +1,86 @@
+package kitchenpos.order.domain;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import kitchenpos.table.domain.OrderTable;
+import org.springframework.util.CollectionUtils;
+
+@Entity
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private OrderTable orderTable;
+    @Enumerated(value = EnumType.STRING)
+    @Column(nullable = false)
+    private OrderStatus orderStatus;
+    @Column(nullable = false)
+    private LocalDateTime orderedTime;
+    @Embedded
+    private final OrderLineItems orderLineItems = new OrderLineItems();
+
+    protected Order() {
+    }
+
+    public Order(OrderTable orderTable, List<OrderLineItem> orderLineItems) {
+        this(null, orderTable, OrderStatus.COOKING, LocalDateTime.now(), orderLineItems);
+    }
+
+    public Order(Long id, OrderTable orderTable, OrderStatus orderStatus, LocalDateTime orderedTime,
+                 List<OrderLineItem> orderLineItems) {
+        validate(orderTable, orderLineItems);
+        this.id = id;
+        this.orderTable = orderTable;
+        this.orderStatus = orderStatus;
+        this.orderedTime = orderedTime;
+        orderLineItems.forEach(this.orderLineItems::add);
+    }
+
+    private void validate(OrderTable orderTable, List<OrderLineItem> orderLineItems) {
+        if (orderTable.isEmpty()) {
+            throw new IllegalArgumentException("빈 테이블은 등록할 수 없습니다.");
+        }
+        if (CollectionUtils.isEmpty(orderLineItems)) {
+            throw new IllegalArgumentException("주문 항목이 1개 이상이어야 합니다.");
+        }
+    }
+
+    public void changeOrderStatus(OrderStatus newOrderStatus) {
+        if (Objects.equals(OrderStatus.COMPLETION.name(), this.orderStatus.name())) {
```

Comment:
enum 은 직접 비교해도 좋을 것 같아요 :)

```suggestion
        if (OrderStatus.COMPLETION == this.orderStatus) {
```

Reply:
수정하겠습니다! 😄